annotator_store/models.py

```diff
@@ -10,25 +10,29 @@
 from django.contrib.auth.models import Group, User
 from django.utils.html import format_html
 from jsonfield import JSONField
-try:
-    import guardian
-    from guardian.shortcuts import assign_perm, get_objects_for_user, \
-        get_objects_for_group, get_perms_for_model, get_perms
-    from guardian.models import UserObjectPermission, GroupObjectPermission
-except ImportError:
-    guardian = None
-import six
-
 
+# get system logger for notifying test errors/warnings
 logger = logging.getLogger(__name__)
-
+#import settings to read ANNOTATION_OBJECT_PERMISSIONS flag
+from django.conf import settings
+#get per-object permission flag
+ANNOTATION_OBJECT_PERMISSIONS = getattr(settings, 'ANNOTATION_OBJECT_PERMISSIONS',False)
+# default guardian installed flag value is false
```

Comment:
I suggest you expand/clarify the comments here to make explicit: only import guardian when object permissions are enabled, warn if it is configured but guardian is not available; note that it defaults to being disabled (rather than "flag value is false", which isn't as meaningful).